kore/src/Kore/Repl/Data.hs

```diff
@@ -825,3 +856,118 @@ substituteAlias
     toString = \case
         SimpleArgument str -> str
         QuotedArgument str -> "\"" <> str <> "\""
+
+createOnePathClaim
+    :: Claim claim
+    => (claim, CommonStrategyPattern)
+    -> Rule.OnePathRule Variable
+createOnePathClaim (cl, cpatt) =
+    Rule.OnePathRule
+    $ Rule.RulePattern
+        { left = pattToLeftPatt cpatt
+        , right = claimToRightPatt cl
+        , requires = Predicate.makeTruePredicate
+        , ensures = claimToEnsures cl
+        , attributes = Default.def
+        }
+  where
+    pattToLeftPatt
+        :: CommonStrategyPattern
+        -> TermLike Variable
+    pattToLeftPatt =
+        toTermLike . fromJust . extractUnproven
```

Comment:
Would it be possible/make sense to extract the pattern from the `CommonStrategyPattern` when we filter the nodes? Since we already know those nodes are not `Bottom` there, so we should be able to limit the amount of unsafe functions (`fromJust`) there where it's obvious. Then we might not even need this helper, we could just take the `TermLike` as an argument in the pair.